mule-gateway/modules/ROOT/pages/mule-gateway-metrics.adoc

```diff
@@ -18,18 +18,25 @@ Even though no sensitive information is collected using this configuration, you
 
 `anypoint.platform.metrics_enabled=false`
 
-== Log analytics event data
+[[log-analytics-event-data]]
+== Log Analytics Event Data
 
-In Mule 4, it is possible to enable API analytics events logging by adding the following entry in the `wrapper.conf` file:
+For Mule 4.6.0 or later, enable `INFO` API analytics event logging to capture data about incoming API requests. 
```

Comment:
It not only from 4.6.0 and later, it also applies for new versions of 4.4 and 4.5